angr/sim_state.py

```diff
@@ -4,16 +4,19 @@
 import weakref
 
 import logging
-l = logging.getLogger(name=__name__)
 
 import angr # type annotations; pylint:disable=unused-import
 import claripy
 import ana
 import archinfo
 
+from archinfo.arch_soot import ArchSoot, SootAddressDescriptor
```

Comment:
Move this line upward by one line (it belongs to the above group).

Reply:
Fixed Now